docs/en/docs/sql-manual/sql-functions/array-functions/array_concat.md

```diff
@@ -24,7 +24,7 @@ under the License.
 
 ## array_concat
 
-<version since="1.2.3">
+<version since="2.0.0">
```

Comment:
For unreleased versions, we prefer to use Next or other tags. Wait until the official release to modify the corresponding version of the document.